e2e/cypress/tests/integration/performance/utils.js

```diff
@@ -1,20 +1,26 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-export const measurePerformance = async (name, upperLimit, cb, runs = 10) => {
+export const measurePerformance = (startMark, upperLimit, endMark = undefined, runs = 5) => {
     return cy.window().its('performance').then(async (performance) => {
-        performance.mark(name);
+        const durations = [];
 
         for (let i = 0; i < runs; i++) {
-            await cb();
-        }
-
-        performance.measure(name);
-        const measure = performance.getEntriesByName(name)[0];
-        const durationMs = Math.round(measure.duration / runs);
+            const startName = `${startMark}_run_${i}`;
+            if (endMark) {
+                const endName = `${endMark}_run_${i}`;
+                performance.mark(endName);
+                performance.measure(startName, endName);
+            }
+            performance.mark(startName);
```

Comment:
(question) Is this measuring anything? Since it marks and measures in the next line, I think it doesn't measure something. Also, the signature of the `measurePerformance` function is used as if the `cb` parameter is there (in the actual tests).